plugins/pitchfork.py

```diff
@@ -25,7 +25,7 @@ def pitchfork(self, msg, arg):
                     @$user, you are being pitchforked down to $place
                     ```
                                                           .+====----->
-                                                           \('
+                                                           \\('
                     =====================================<%{%{%{>>+===---> $user
                                                            /(,
```

Comment:
Make this like `//(,` just to make the trident symmetric?

Reply:
Yeah this could be done as well , but is it really required :P

Reply:
Yes.

Reply:
Actually it has to be just here. The double escape will in the chat print as a single one, so making below a double `//` would then look weird in the output. You could probably just prepend `r` to the string to make it a raw string so it keeps symmetric in code and in output.

Reply:
Yeah your probably right, I will prepend r to it so that it makes the code symmetric.